.gitignore

```
@@ -0,0 +1 @@
+.DS_Store
```

I would add the normal PHP staples here
```suggestion
.DS_Store
vendor/
composer.lock
```